<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dadaPhoto</class>
 <widget class="QMainWindow" name="dadaPhoto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dadaPhoto</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressources.qrc">
    <normaloff>:/images/dadaphoto.png</normaloff>:/images/dadaphoto.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="listWidget">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_nombre">
          <property name="text">
           <string>Pas de photos</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="zoneImage">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Désolé…  Le dossier est vide :(</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_valider">
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>75</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>125</width>
            <height>75</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Valider</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/boutons/images/appliquer.png</normaloff>:/boutons/images/appliquer.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>36</width>
            <height>36</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>2</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_antihoraire">
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>125</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Antihoraire</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/boutons/images/rotation_antihoraire.png</normaloff>:/boutons/images/rotation_antihoraire.png</iconset>
          </property>
          <property name="shortcut">
           <string>Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_horaire">
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>125</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Horaire</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/boutons/images/rotation_horaire.png</normaloff>:/boutons/images/rotation_horaire.png</iconset>
          </property>
          <property name="shortcut">
           <string>Down</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_imprimer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>125</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Imprimer</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/boutons/images/imprimer.png</normaloff>:/boutons/images/imprimer.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_supprimer">
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>125</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Supprimer</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/boutons/images/supprimer.png</normaloff>:/boutons/images/supprimer.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>2</width>
            <height>150</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_suivant">
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>125</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Suivante</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/boutons/images/next.png</normaloff>:/boutons/images/next.png</iconset>
          </property>
          <property name="shortcut">
           <string>Right</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_precedent">
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>125</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Précédente</string>
          </property>
          <property name="icon">
           <iconset resource="ressources.qrc">
            <normaloff>:/boutons/images/previous.png</normaloff>:/boutons/images/previous.png</iconset>
          </property>
          <property name="shortcut">
           <string>Left</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir_un_dossier"/>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionImporter"/>
    <addaction name="actionNettoyer"/>
    <addaction name="actionRefreshList"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="action_propos"/>
    <addaction name="action_propos_de_Qt"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/menus/images/quitter.png</normaloff>:/menus/images/quitter.png</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="action_propos">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/menus/images/about.png</normaloff>:/menus/images/about.png</iconset>
   </property>
   <property name="text">
    <string>À propos</string>
   </property>
  </action>
  <action name="actionOuvrir_un_dossier">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/menus/images/ouvrir.png</normaloff>:/menus/images/ouvrir.png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir un dossier</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/menus/images/enregistrer.png</normaloff>:/menus/images/enregistrer.png</iconset>
   </property>
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
  <action name="action_propos_de_Qt">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/menus/images/qt.png</normaloff>:/menus/images/qt.png</iconset>
   </property>
   <property name="text">
    <string>À propos de Qt</string>
   </property>
  </action>
  <action name="actionImporter">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/menus/images/import.png</normaloff>:/menus/images/import.png</iconset>
   </property>
   <property name="text">
    <string>Importer</string>
   </property>
  </action>
  <action name="actionNettoyer">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/menus/images/nettoyer.png</normaloff>:/menus/images/nettoyer.png</iconset>
   </property>
   <property name="text">
    <string>Nettoyer</string>
   </property>
  </action>
  <action name="actionRefreshList">
   <property name="text">
    <string>Rafraîchir la liste</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
